dp/generate_climos.py

```diff
@@ -366,10 +370,29 @@ def update_metadata_and_time_var(input_file, t_start, t_end, climo_filepath):
             raise ValueError('Expected climo file to contain # time values in {}, but found {}'
                              .format(num_times_to_interval_set.keys(), cf.time_var.size))
 
+
+        # Update cell_methods to reflect the operation being done to the data
+        cell_method_op = ''
+        if operation == 'std':
+            cell_method_op = 'standard_deviation'
+        elif operation == 'mean':
+            cell_method_op = operation
+
+        for key in cf.variables.keys():
+            try:
+                cf.variables[key].cell_methods = cf.variables[key].cell_methods + ' time: {} over days'.format(cell_method_op)
+            except AttributeError as e:
+                # skip over vars that do not have cell_methods i.e. lat, lon
+                continue
+
         # Update frequency attribute to reflect that this is a climo file.
+        suffix = ''
+        if operation == 'std':
+            suffix = 'SD'
+
```

Comment:
Also, I think we should start using the suffix 'Mean' explicitly here, instead of continuing the past practice of not stating the operation and assuming mean in that case. This will go into the PCIC Metadata Standard when I get to it. @jameshiebert do you agree?

Reply:
Yeah, I'd second that motion.

Reply:
Wasn't the purpose of leaving it to ensure backwards compatibility?  Do we have to go back and change all the files to have `xClimmean`?

Reply:
No, the idea is to go forward with being explicit about mean (`xClimMean`, note capital M) from here on in. In the case the operator is not specified (`xClim`), it is assumed to be mean. With that convention, we don't have to change any existing files, but we are explicit with all files created from here on in. It's a common backwards-compatibility strategy.